apps/wasm-standalone/wasm-runtime/Cargo.toml

```diff
@@ -26,8 +26,8 @@ license = "Apache-2.0"
 keywords = ["wasm", "machine learning", "wasmtime"]
 
 [dependencies]
-wasmtime = "0.16.0"
-wasmtime-wasi = "0.16.0"
+wasmtime = "0.28.0"
```

Comment:
Can u check it if this version of `wasmtime` supports loading wasm graph with `-mattr=+simd128` flag? Thanks!

Reply:
> Can u check it if this version of `wasmtime` supports loading wasm graph with `-mattr=+simd128` flag? Thanks!

Yes. I've already tested it. `wasmtime` 0.28 supports it.